release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -600,6 +736,10 @@ service class EchoService {
 
 ###### All the timeout configurations are converted to accept decimal values and the time unit is in seconds.
```

Comment:
```suggestion
###### All the timeout configurations are converted to accept decimal values and the time unit is in seconds.
```

Complete sentences shouldn't be added as titles.

Reply:
I fixed it with https://github.com/ballerina-platform/ballerina-release/pull/115